### DISCUSSION TITLE
请问现在 Dapp 开发支持 Filecoin 签名吗？

### DISCUSSION THREAD

**Discussion by stonega** [consenlabs/fe-public-template#2] [Q&A]
您好，请问现在 Dapp 开发支持 Filecoin 签名吗？

**Comment by kaichen** [comment]
filecoin 社区有这类 dapp 接入标准了吗？

**Reply by stonega** [reply]
[Mestamsksnap](https://github.com/MetaMask/metamask-snaps-beta) 可以通过[插件](https://filecoin-shipyard.github.io/filecoin.js/docs/setup-metamask-provider)支持 Flecoin签名，目前似乎只有这种方案。
